include/UIKit/UIKitTypes.h

```diff
@@ -48,7 +48,7 @@ typedef NS_ENUM(NSUInteger, NSWritingDirection) {
 };
 
 enum {
-    UITextAlignmentLeft,
+    UITextAlignmentLeft = 0,
     UITextAlignmentCenter,
```

Comment:
or the other way around actually.. yeah

---
In reply to: [85629574](https://github.com/Microsoft/WinObjC/pull/1254#discussion_r85629574) [](ancestors = 85629574,85553544)